### PR TITLE
CTC-44: view lesson guides on lesson overview page

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

Music year: 1975

- Adds lesson guide to the lesson overview
- Utilises new lesson content mv
- Bumps oak-curriculum-schema package version
- Updates schema to remove query schemas to use the 'old' curriculum schema

### Acceptance Criteria:

- [x]  If a lesson has a lesson guide (google doc), display it at the top of the lesson resources
- [x]  Go with option 1 from here: https://github.com/oaknational/Oak-Web-Application/pull/2409
    - Do locked things suffer from the same cursor issue mentioned on this PR?
- [x]  No reference to download, that can come later

Fixes #CTC-44

## How to test

1. Go to https://deploy-preview-2973--oak-web-application.netlify.thenational.academy/teachers/beta/lessons/running-as-a-team
2. Lesson guide is viewable on the beta lesson overview page (Not downloadable as of yet)

## Screenshots

How it should now look:

Screenshot 2024-11-08 at 15 45 09

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx [hunk 1]
```diff
@@ -274,6 +283,50 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
...
               <OakFlex $flexDirection={"column"} $position={"relative"}>
+                {pageLinks.find((p) => p.label === "Lesson guide") &&
+                  lessonGuideUrl && (
+                    <LessonItemContainer
+                      isSpecialist={isSpecialist}
+                      ref={lessonGuideSectionRef}
+                      title={"Lesson guide"}
+                      //Defaulted to false until download ticket implementation
+                      downloadable={false}
+                      // Avo types need to be updated to include lesson guide
```
Comment (njblore) [inline src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:294]: do we have a ticket for this?

File: src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx [hunk 2]
```diff
@@ -274,6 +283,50 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
...
+                          <iframe
+                            data-testid="lesson-guide-iframe"
+                            src={`${previewLessonGuideUrl}`}
+                            title={`lesson guide: ${lessonTitle}`}
+                            width="auto"
+                            height="100%"
+                            style={{
+                              border: "none",
+                            }}
+                            //small render bug fix to make sure the iframe assumes 100% width. Docs are still in unpublished currently so temporary
```
Comment (njblore) [inline src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:319]: do we have these todos captured in a ticket?

**Review by njblore — APPROVED** [review]
looking good 😎 just make sure we've got all the tidy up work captured outside the code

File: src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx (hunk 1 above)
Comment (k-huggs) [reply src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:294]: Yeah working on it atm https://www.notion.so/oaknationalacademy/12926cc4e1b18015a518efe15426e53b?v=ab3cc64052624acaa90f016720b46001&p=12f26cc4e1b180b9954bfaf991d2416f&pm=s

File: src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx (hunk 2 above)
Comment (k-huggs) [reply src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:319]: Will create a ticket for this now

**Review by Monika-Koziol — COMMENTED** [review]
All looking good. Just wanted to check on one element: is the black border the same thickness as the border around other assets? It looks somewhat thinner.

**Comment by RBrownsett** [comment]
@k-huggs looking good - I did notice that when the connection is poor the guide loads on half of the frame before expanding. Seems to happen in Safari more often. Can it be set to take up all of the frame? 


https://github.com/user-attachments/assets/88a5fcee-df4d-4fa9-8562-d38efcf84552

**Comment by k-huggs** [comment]
> All looking good. Just wanted to check on one element: is the black border the same thickness as the border around other assets? It looks somewhat thinner.

Updated the border thickness to match other assets ✅

**Comment by k-huggs** [comment]
> @k-huggs looking good - I did notice that when the connection is poor the guide loads on half of the frame before expanding. Seems to happen in Safari more often. Can it be set to take up all of the frame?
> 
>  Screen.Recording.2024-11-12.at.17.13.13.mov

This is a known bug, I believe it's due to the documents being in edit mode. I've made a ticket to check this https://www.notion.so/oaknationalacademy/I-want-Lesson-guides-to-render-correctly-13c26cc4e1b180d5ba2bf7acde52e4c1

**Comment by k-huggs** [comment]
> @k-huggs looking good - I did notice that when the connection is poor the guide loads on half of the frame before expanding. Seems to happen in Safari more often. Can it be set to take up all of the frame?
> 
>  Screen.Recording.2024-11-12.at.17.13.13.mov

This is a known issue, I think it could be something to do with the docs sill being in edit mode and not published. I've created  a ticket to check for this https://www.notion.so/oaknationalacademy/I-want-Lesson-guides-to-render-correctly-13c26cc4e1b180d5ba2bf7acde52e4c1

**Comment by Monika-Koziol** [comment]
@k-huggs 4px is too strong, so let's go down to 2px so the border doesn't draw attention away from the content. Thanks for making this update.

**Review by Monika-Koziol — APPROVED** [review]

**Comment by k-huggs** [comment]
@RBrownsett I've published this document and it's possible to tab through it. There are some issues with the formatting on the documents not sure if this is due to the aspect ratio or the fact the document is unfinished 
https://github.com/user-attachments/assets/9e010719-6b47-48ea-b18b-2ee0c30911c5

**Review by RBrownsett — APPROVED** [review]
Approved - tested across desktop and mobile. After paired testing session with Kyle it was possible to tab through into the Frame when the document was published

**Pull request merged**